### DISCUSSION TITLE
Source Code Without Debug and Log Options

### DISCUSSION THREAD

**Discussion by MrAshFar** [grumpycoders/pcsx-redux#704] [Ideas]
Hello, I'm using this source code. However, it has a lot of extra options like debug and log. I would not able to remove these options from the source and tightly coupled to source code. is it possible to have another source code with doesn't contain debug and log?

**Comment by wheremyfoodat** [comment]
PCSX-Redux's goal is, first and foremost, to provide a good experience to developers, which is why there's so many debugging tools integrated into the source code. Any particular reason you want a version of the code wiith them purged from existence?

**Comment by MrAshFar** [comment]
"Any particular reason you want a version of the code with them purged from existence?"

Yes, I want to use this source code with a minimal change to end-users (mobile, desktop, and portable devices) who don't like extra options that confuse them. in other words, select game set graphic configuration and start the game.

And also, it causes execution overheads because of logs and debugs. Yes, of course, it is useful for developers but not for end-users.

**Reply by NDR008** [reply]
I think there are emulators in a better state for your goals.
